sales.html

```diff
@@ -12,6 +12,35 @@
   <div id="content_wrapper">
     <h2>Sales Projections</h2>
     <table id="sales" class="data"></table>
+<form id="new_store_form">
+  <fieldset>
+    <label for="storeName">Store Name</label>
+    <input id="storeName" type="text" name="storeName">
+
+    <label for="identifier">Store ID</label>
+    <input id="identifier" type="text" name="identifier">
+
+    <label for="storeAddress">Store Address</label>
+    <input id="storeAddress" type="text" name="storeAddress">
+
+    <label for="storePhone">Store Phone</label>
+    <input id="storePhone" type="tel" name="storePhone">
+
+    <label for="minCustsPerHour">Minimum Number of Customers Per Hour</label>
+    <input id="minCustsPerHour" type="number" name="minCustsPerHour">
+
+    <label for="maxCustsPerHour">Maximum Number of Customers Per Hour</label>
+    <input id="maxCustsPerHour" type="number" name="maxCustsPerHour">
+
+    <label for="avgCookiesPerCust">Average Cookies Sold Per Customer</label>
+    <input id="avgCookiesPerCust" type="number" name="avgCookiesPerCust">
+
+    <input type="submit">
+
+  </fieldset>
+
+</form>
+
     <footer>
```

Comment:
@AlisonZ @elaewin this should be indented to the left and include the same level of indentation as the form element